db/model.js

```diff
@@ -12,10 +12,23 @@ module.exports.fetchArticles = function() {
   return db.query("SELECT * FROM articles ORDER BY created_at DESC;")
 }
 
-module.exports.fetchCommentsByArticle = function(id) {
-  const sql = `SELECT * FROM comments WHERE article_id = ${id};`
+module.exports.fetchCommentsByArticle = function(id, internal = false) {
+  const sql = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`
   //console.log(sql)
-  return db.query(sql)
+  if (/^\d+$/.test(id)) {
```

Comment:
What does psql do when you try to pass it an invalid article id? Could we take advantage of this instead of trying to manually check this ourselves? As a rule of thumb, always try making the request and if psql gives you an error to work with, use it - and if it doesn't but you expect to send an error then that's when you need a custom one!